PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/database/component/PasswordDialog.java

```diff
@@ -58,10 +60,17 @@ public PasswordDialog(Project project, DatabaseResource resource) {
         testConnectionActionPanel.setVisible(false);
         testResultTextPane.setEditable(false);
         testResultTextPane.setText(StringUtils.EMPTY);
-        final Dimension lastColumnSize = new Dimension(106, 30);
-        passwordSaveComboBox.setPreferredSize(lastColumnSize);
-        passwordSaveComboBox.setMaximumSize(lastColumnSize);
-        passwordSaveComboBox.setSize(lastColumnSize);
+        final Dimension lastColumnSize = new Dimension(106, this.passwordSaveComboBox.getPreferredSize().height);
+        this.passwordSaveComboBox.setPreferredSize(lastColumnSize);
+        this.passwordSaveComboBox.setMaximumSize(lastColumnSize);
+        this.passwordSaveComboBox.setSize(lastColumnSize);
+        if (Objects.nonNull(resource.getPassword()) && Objects.nonNull(resource.getPassword().saveType())) {
+            this.passwordSaveComboBox.setValue(resource.getPassword().saveType());
+        } else {
+            this.passwordSaveComboBox.setValue(Arrays.stream(Password.SaveType.values())
+                    .filter(e -> StringUtils.equals(e.name(), AzureConfigurations.getInstance().passwordSaveType())).findAny()
+                    .orElse(Password.SaveType.UNTIL_RESTART));
+        }
```

Comment:
is this issue important/severe enough to be fixed as a hotfix?

Reply:
should modify in `setData`
https://github.com/microsoft/azure-tools-for-java/pull/5622/files#diff-6005d74a14cff22bf7a458fab4d67ded17c3e7112649afe323ccf6691e01b1dcR161-R163

Reply:
updated.